src/library_nodepath.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The Emscripten Authors
+ * SPDX-License-Identifier: MIT
+ */
+
+mergeInto(LibraryManager.library, {
+  $PATH: {
+    isAbs: (path) => nodePath['isAbsolute'](path),
+    normalize: (path) => nodePath['normalize'](path),
+    dirname: (path) => nodePath['dirname'](path),
+    basename: (path) => nodePath['basename'](path),
+    join: function () {
+      return nodePath['join'].apply(null, arguments);
+    },
```

Comment:
Can this just be `join: nodePath['join']`?   (same with above)

Reply:
I tried that initially, but it caused the acorn-optimizer to fail.